doc/src/manual/faq.md

```diff
@@ -980,3 +980,11 @@ If this describes you, you may also be interested in reading our [guidelines for
 
 Links to each of these download types can be found on the download page at [https://julialang.org/downloads/](https://julialang.org/downloads/).
 Note that not all versions of Julia are available for all platforms.
+
+### How can I transfer the list of installed packages after updating my version of Julia?
+
+When you add a package, this is not really done for the version of Julia that is being used, but for a certain [environment](https://docs.julialang.org/en/v1/manual/code-loading/#Environments-1). However, each minor version of Julia has its own default environment, defined by the files `Project.toml` and `Manifest.toml` in the folder `.julia/environments/vX.Y`, where `X.Y` is the number of the version. The packages defined in those files are the ones that can normally be used when you start a session in that version of Julia.
```

Comment:
The first sentence is a bit of a pronoun mess. How about starting with the fact about the environments and then elaborating on the implications:

```
By default each minor version of julia has its own default [environment](...). As a result, upon installing a new minor version of Julia, the packages you added using the previous minor version will not be available by default. The default environment for a given julia version is defined by ...
```

Reply:
Done, thanks for the suggestion